code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -176,6 +180,15 @@
 				update = TRUE
 		if(update)
 			host_mob.update_damage_overlays()
+		if(C.getStaminaLoss() < 100) //Stops after hitting the second slowdown level. This is distributed so only ~80% is counted towards crit
```

Comment:
damage get procs use % to crit total not actual damage so this will actually stamcrit people

Reply:
👍